lsix

```diff
@@ -213,7 +213,7 @@ main() {
 
 
     # Resize on load: Save memory by appending this suffix to every filename.
-    resize="[${tilewidth}x${tileheight}]"
+    #resize="[${tilewidth}x${tileheight}]"  ### DONE: SC2034 (warning): resize appears unused. Verify use (or export if used externally).
```

Comment:
This is a good point. I should probably just remove the whole thing as it was causing problems (with animated GIFs, if I recall correctly).

Reply:
I removed these unused lines.